streaming-binlog/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,26 @@
+#// Copyright 2020 Google LLC
```

Comment:
nit: I think `//` is redundant (we should actually add a maven plugin to fix these license messages automatically for us, this is my bad).

Reply:
Nice catch! I'll format them correctly. I have also created a ticket to do this!